assistant/prompt/v0.1.0/schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://common.schemas.verida.io/assistant/prompt/v0.1.0/schema.json",
+  "name": "https://common.schemas.verida.io/assistant/prompt/v0.1.0/schema.json",
+  "title": "User Prompt",
+  "titlePlural": "User Prompts",
+  "description": "Saved prompts for the assistant",
+  "type": "object",
+  "database": {
+    "name": "prompts"
+  },
+  "allOf": [
+    { "$ref": "https://core.schemas.verida.io/base/v0.1.0/schema.json" },
+    {
+      "properties": {
```

Comment:
If `name` is used, you can redefine it with an updated description explaining the field.